addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js

```diff
@@ -156,6 +150,34 @@ odoo.define('point_of_sale.ProductScreen', function(require) {
 
             return { draftPackLotLines, quantity: weight, description, price_extra };
         }
+        async _openProductConfiguratorPopup(product) {
+            let attributes = _.map(product.attribute_line_ids, (id) => this.env.pos.attributes_by_ptal_id[id])
+                                  .filter((attr) => attr !== undefined);
+
+            // avoid opening the popup when each attribute has only one available option.
+            if (_.some(attributes, (attribute) => attribute.values.length > 1)) {
```

Comment:
you forgot to take into account the custom attribute. Even if there is only one value for the attribute but that value is custom, it should show the popup.

Reply:
@caburj Thanks for your review.
I modified the code.